src/lib/api.js

```diff
@@ -211,3 +216,35 @@ export const getPDFReport = (userId, email) => {
     mode: 'cors'
   });
 };
+
+// only make network request for readers when we have none
+// populateReaders (dispatch: Function, getState: Function) => Promise
+const populateReaders = (dispatch, getState) => {
+  const readersCount = getReaders(getState()).size;
+  return readersCount ? Promise.resolve : readersRequest()(dispatch, getState);
```

Comment:
I see the function being used in a `promise.all()` further down in the file. Should `Promise.resolve` be `Promise.resolve()` ?